lib/writeResources.js

```diff
@@ -112,13 +115,26 @@ function writeDataUri(object, extension) {
     object.uri = 'data:' + mimeType + ';base64,' + source.toString('base64');
 }
 
-function writeBufferView(gltf, object) {
+function writeBufferView(gltf, object, writtenResourceMap) {
     delete object.uri;
+
+    // If we've written this resource before, re-use the bufferView
+    const resourceId = object.extras._pipeline.resourceId;
+    if (defined(resourceId) && defined(writtenResourceMap[resourceId])) {
```

Comment:
From `readResources.js` it looks like `resourceId` isn't supposed to be defined here if `writtenResourceMap` isn't refined, but should this also check `defined(writtenResourceMap)`?